metpy/calc/kinematics.py

```diff
@@ -231,7 +231,7 @@ def shearing_deformation(u, v, dx, dy):
 
     See Also
     --------
-    stretching_convergence, shearing_stretching_deformation
```

Comment:
Could you instead of deleting change to "stretching_deformation" and also add "shear_deformation" here?

Reply:
I've corrected that to be stretching_deformation instead of deleting the stretching_convergence reference. Although, would you be able to mention what "shear_deformation" should be, as this is within the shearing_deformation function? Thanks!

Reply:
Oh...oops. 😁

Reply:
Sounds good :+1: